includes/admin/class-amp-editor-blocks.php

```diff
@@ -43,8 +43,16 @@ public function init() {
 		if ( function_exists( 'gutenberg_init' ) ) {
 			add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_block_editor_assets' ) );
 			add_filter( 'wp_kses_allowed_html', array( $this, 'whitelist_block_atts_in_wp_kses_allowed_html' ), 10, 2 );
-			add_filter( 'the_content', array( $this, 'tally_content_requiring_amp_scripts' ) );
-			add_action( 'wp_print_footer_scripts', array( $this, 'print_dirty_amp_scripts' ) );
```

Comment:
Do you think the methods `tally_content_requiring_amp_scripts()` and `print_dirty_amp_scripts()` could be removed, given that they're not used anymore? Maybe the [comment](https://github.com/Automattic/amp-wp/pull/1199/files#diff-af86f8db0fb2fe24a8aea92de5ee5128R48) below could [link to them](https://github.com/Automattic/amp-wp/blob/5f635d27d64f50eeba4c0710c84d929342f967b5/includes/admin/class-amp-editor-blocks.php#L156-L176).

Reply:
You're probably right. It's easy enough to look back in history to find the removal of these methods to resurrect them. 👍